doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc

```diff
@@ -0,0 +1,28 @@
+[id='configuring-emails-in-task-notification-proc']
+
+= Configuring emails in task notification
+
+Earlier it was possible to send notifications only to users or group of users in {CENTRAL}. Now you can directly add any email addresses as well.
+
+.Prerequisites
+
+You have created a project in {CENTRAL} and it contains at least one business process asset.
+
+.Procedure
+
+. Create a business process.
+. In the process designer, select the *Activities* tool from the tool palette.
+. Select *User*.
+. Drag and drop a user task onto the process designer canvas.
+. In the upper-right corner of the screen, click the *Properties* icon.
+. Expand *Implementation/Execution* and next to *Notifications*, click image:getting-started/btn_assign.png[] to open the *Notifications* window.
+. Click *Add*.
+. In the *Notifications* window, enter an email address in the *To: email(s)* field to send task notification emails.
```

Comment:
"to send task notification emails." here doesn't sound clear. I would change it to something like: "to set the recipients of the task notification emails ...".

Reply:
Done.